druid-shell/src/backend/windows/window.rs

```diff
@@ -1393,30 +1396,30 @@ impl WindowBuilder {
             let mut dwStyle = WS_OVERLAPPEDWINDOW;
             let mut dwExStyle: DWORD = 0;
             let mut focusable = true;
+            let mut parent_hwnd = None;
+            let window_level = RefCell::new(self.level.clone());
             if let Some(level) = self.level {
                 match level {
                     WindowLevel::AppWindow => (),
-                    WindowLevel::Tooltip(parent_window_handle) => {
+                    WindowLevel::Tooltip(parent_window_handle)
+                    | WindowLevel::DropDown(parent_window_handle) => {
```

Comment:
Maybe `Modal` here too? In principle, this "regresses" the topmost behavior of modal, but I'm pretty sure the current code is dead code anyway, because `set_level` doesn't currently set modal.

Reply:
Added `Modal`